Example/Example/ComicsViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ComicsViewController.swift
+//  Example
+//
+//  Created by Davide Mendolia on 19/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import BothamUI
+
+class ComicsViewController: BothamViewController<ComicsPresenter>, BothamCollectionViewController, ComicsUI, UICollectionViewDelegateFlowLayout {
+
+    @IBOutlet var collectionView: UICollectionView!
+    var dataSource: BothamCollectionViewDataSource<Comic, ComicCollectionViewCell>!
```

Comment:
It cant because it's part of the protocol. It should be at least the same visibilty that it's container. in this case ComicsViewController is internal.